src/test/java/in/zapr/druid/druidry/extensions/datasketches/postAggregator/ThetaSketchEstimatePostAggregatorTest.java

```diff
@@ -73,6 +73,30 @@ public void testAllFieldsWithFieldAccess() throws JsonProcessingException, JSONE
 
     }
 
+    @Test
+    public void testAllFieldsWithFieldAccess() throws JsonProcessingException, JSONException {
+
+        FieldAccessPostAggregator fieldAccessPostAggregator =
+                new FieldAccessPostAggregator("stars");
+
+        ThetaSketchEstimatePostAggregator thetaSketchEstimatePostAggregator =
```

Comment:
You need to update all the test cases with this new field and with constructor changed to builder.

Reply:
Approving and merging into separate feature branch to fix test cases.